src/app/components/modules/LoginForm.jsx

```diff
@@ -211,12 +211,11 @@ class LoginForm extends Component {
                         {tt('g.cancel')}
                     </button>}
                 </div>
-                {authType == 'Posting' &&
```

Comment:
this is great because authType is defined as the result of a call to tt, which i'm pretty sure will return a translated string -- there's no way of knowing this will be `Posting` in the first place, depending on the locale.